corral/util/error.pony

```diff
@@ -1,4 +0,0 @@
-class val Error
```

Comment:
was this not used?

Reply:
I removed it because it clashed with logger.Error, and it only ever held strings to be passed to a logger.

Reply:
Makes sense.